ansible/vars/icds-new/icds-new_public.yml

```diff
@@ -193,7 +193,7 @@ nameservers:
   - 164.100.17.3
   - 164.100.3.1
 
-http_proxy_address: '10.247.63.132'
+http_proxy_address: '10.247.24.16'
```

Comment:
this could also eventually just be 
```
http_proxy_address: {{ groups.squid.0 }}
```
if we start setting that group correctly, or even just
```
http_proxy_address: {{ groups.patch0 }}
```
right?

Reply:
correct